### DISCUSSION TITLE
How to manage the export HTML file body background?

### DISCUSSION THREAD

**Discussion by tianfanghan** [marp-team/marp#183] [Q&A] [answered]
I created my own theme, and import the default theme. When I export a html file, the html body background always be black, so I wonder if I can change the default body background color?
(NOT section background color.)

**Comment by yhatt** [comment]
Exactly same as https://github.com/marp-team/marp-cli/issues/358#issuecomment-862041612. Marp does not have an ability to control styles for the outside of the slide, such as `<body>`.

**Comment by tianfanghan** [comment]
I didn't understand the link https://github.com/marp-team/marp-cli/issues/358#issuecomment-862041612 says. But it's OK, you told me it's not possible, I can edit the HTML file manually. That's more efficient. But thank you very much for your time.

**Comment by tianfanghan** [comment]
Bye the way, thanks a lot for the beautiful work, we love it.

**Comment by yhatt** [comment, marked as answer]
**UPDATE:** The latest [Marp CLI v1.5.0](https://github.com/marp-team/marp-cli/releases/tag/v1.5.0) follows [the update of the base framework Marpit](https://github.com/marp-team/marpit/releases/tag/v2.2.0). It can style the slide container through [`::backdrop` CSS selector](https://developer.mozilla.org/docs/Web/CSS/::backdrop).
https://marpit.marp.app/inline-svg?id=backdrop-css-selector

Now you can change the black backdrop via CSS (working in both of theme CSS and Markdown inline style):

```html
<style>
::backdrop {
  background-color: #448;
}
</style>
```

**Reply by tianfanghan** [reply]
amazing
